code/edgemedia/build.gradle.kts

```diff
@@ -31,21 +31,10 @@ aepLibrary {
 }
 
 dependencies {
-    // Stop using snapshots after core, edge and edge identity release.
-    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
-    implementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion-SNAPSHOT"){
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-        exclude(group = "com.adobe.marketing.mobile", module = "edgeidentity")
-    }
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")
+    implementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion")
 
-    androidTestImplementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion-SNAPSHOT"){
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-        exclude(group = "com.adobe.marketing.mobile", module = "edgeidentity")
-    }
-    // Update it to 3.+ after release
-    androidTestImplementation("com.adobe.marketing.mobile:edgeidentity:3.0.0-SNAPSHOT"){
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-    }
+    androidTestImplementation("com.adobe.marketing.mobile:edgeidentity:3.0.0")
```

Comment:
for androidTestImplementation should this be edge and edgeidentity?

Reply:
Edge is a dependency for the project, so it's automatically a dependency to the tests as well.

Reply:
are there any errors if we explicitly specify that?

Reply:
I don't see any error when include
 androidTestImplementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion")